<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userDialog</class>
 <widget class="QDialog" name="userDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>391</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>62</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>62</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>水费</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>电费</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>煤气费</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>14</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>14</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>从</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>14</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>14</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>到</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_2">
         <property name="minimumSize">
          <size>
           <width>96</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>当前余额 :  元</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>详细信息</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>411</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>251</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>231</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>路灯</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>广播</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>其他</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>设备名 : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>描述   : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>状态   : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>需要维修</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>正在维修</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>已经维修</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>日期   : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_3">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string>上传</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_11">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
